docs/manual/manual_safety_systems.rst

```diff
@@ -0,0 +1,255 @@
+Microcontroller temperature derate is calculated using the module's current microcontroller temperature, Temperature Monitor Microcontroller's ``otw`` parameter 
+(over temperature warning), and Temperature Monitor Microcontroller's ``otlo`` parameter (over temperature lock out). You can find the over temperature parameters 
+listed in the :ref:`Temperature Monitor Microcontroller Message Table <temperature_monitor_microcontroller>`.
+
+The total microcontroller temperature derate is calculated by :math:`\text{Microcontroller Temperature Derate} = \frac{\text{otlo - temp}}{\text{otlo - otw}}` where the solution is saturated to [0,1].
+
+This means that for any temperature below otw, there is no derate applied. For any temperature at or above otlo, 
+the microcontroller temperature derate is 0. Any microcontroller temperature in that range is subject to a variable derate [0, 1].
+
+Suppose your module's otw is 100°C and its otlo is 110°C. This means that there will be no derate limiting applied under 100°C, your module will stop spinning at and above 110°C, and will be derate limited in between.
+
+A graph illustrating the microcontroller temperature derate calculation is provided below:
+
+.. image:: ../_static/manual_images/safety/micro_temp_derate_graph.png
+    :align: center
+    :scale: 50%
+
+Coil Temperature Based Derate
+---------------------------------
+As mentioned above, your module's coil temperature derate is meant to protect your module's coils from overheating. Coil temperature derate is calculated using the 
+module's current estimated coil temperature, Coil Temperature Estimator's ``otw`` parameter (over temperature warning), and Coil Temperature Estimator's ``otlo`` 
+parameter (over temperature lock out). You can find the over temperature parameters listed in the :ref:`Coil Temperature Estimator Message Table <coil_temperature_estimator>`.
+
+The total coil temperature derate is calculated by :math:`\text{Coil Temperature Derate} = \frac{\text{otlo - temp}}{\text{otlo - otw}}` where the solution is saturated to [0,1].
+
+This means that for any temperature below otw, there is no derate applied. For any temperature above otlo, the microcontroller 
+temperature derate is 0. Any microcontroller temperature in that range is subject to a variable derate [0, 1].
+
+Suppose your module's otw is 140°C and its otlo is 150°C. This means that there will be no derate limiting applied under 140°C, your module will stop spinning at and above 150°C, and will be derate limited in between.
+
+A graph illustrating the microcontroller temperature derate calculation is provided below:
+
+.. image:: ../_static/manual_images/safety/coil_temp_derate.png
+    :align: center
+    :scale: 50%
+
+Calculation of the Total Derate
+-----------------------------------
+The total applied derate applied is the direct multiplication of the speed, microcontroller, and coil derate values. So, then our drive voltage system becomes:
```

Comment:
This has applied twice in the first bit, probably only needs it once, the part I mean is "The total applied derate applied is...."